database/migration/database_schema_registration_06_29_2016 .sql

```diff
@@ -30,17 +30,17 @@ CREATE TABLE IF NOT EXISTS `address` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
   `employee_id` int(11) unsigned NOT NULL,
   `type` enum('residence','office') COLLATE utf8_bin NOT NULL,
-  `phone` int(11) DEFAULT NULL,
-  `fax` bigint(12) DEFAULT NULL,
+  `phone` varchar(10) COLLATE utf8_bin DEFAULT NULL,
+  `fax` varchar(11) COLLATE utf8_bin DEFAULT NULL,
   `street` varchar(100) COLLATE utf8_bin DEFAULT NULL,
-  `pin_no` int(7) DEFAULT NULL,
+  `pin` varchar(6) COLLATE utf8_bin DEFAULT NULL,
```

Comment:
If you are sure pin would not be less than 6 then can use char instead of varchar